bundles/io/org.eclipse.smarthome.io.transport.mqtt/src/main/java/org/eclipse/smarthome/io/transport/mqtt/reconnect/AbstractReconnectPolicy.java

```diff
@@ -22,10 +22,21 @@
      *
      * @param brokerConnection The broker connection
      */
-    void setBrokerConnection(MqttBrokerConnection brokerConnection) {
+    public void setBrokerConnection(MqttBrokerConnection brokerConnection) {
         this.brokerConnection = brokerConnection;
     }
 
+    /**
+     * Return the brokerConnection object that this reconnect policy is assigned to.
+     */
+    public MqttBrokerConnection getBrokerConnection() {
+        return brokerConnection;
+    }
+
+    public boolean isReconnecting() {
```

Comment:
Is this also a candidate for being `abstract`? This would force the implementations to think about what to return here.